docs/src/integrations/exchange.md

```diff
@@ -0,0 +1,183 @@
+# Add Solana to Your Exchange
+
+This guide describes how to add Solana's native token SOL to your cryptocurrency
+exchange.
+
+## Node Setup
+
+We highly recommend setting up at least one of your own Solana api nodes to
+give you a trusted entrypoint to the network and allow you full control over how
+much data is retained.
+
+To run an api node:
+1. [Install the Solana command-line tool suite](../cli/install-solana-cli-tools.md)
+2. Boot the node with at least the following parameters:
+```bash
+solana-validator \
+  --ledger <LEDGER_PATH> \
+  --entrypoint <CLUSTER_ENTRYPOINT> \
+  --expected-genesis-hash <EXPECTED_GENESIS_HASH> \
+  --expected-shred-version <EXPECTED_SHRED_VERSION> \
+  --rpc-port 8899 \
+  --no-voting \
+  --enable-rpc-transaction-history
+```
+
+  Customize `--ledger` to your desired ledger storage location, and `--rpc-port` to the port you want to expose.
+
+  The `--entrypoint`, `--expected-genesis-hash`, and `--expected-shred-version` parameters are all specific to the cluster you are joining.
+  [Current parameters for Mainnet Beta](../clusters.md#example-solana-validator-command-line-2)
+
+  Optional parameters to consider:
+  - `--limit-ledger-size` specifies how many ledger shreds to retain on disk. If you do not include this parameter, the ledger will keep the entire ledger until it runs out of disk space. A larger value like `--limit-ledger-size 250000000000` is good for a couple days
+  - `--trusted-validator` can protect you from booting from a malicious snapshot. [More on the value of booting with trusted validators](../running-validator/validator-start.md#trusted-validators)
+
+## Setting up Deposit Accounts
+
+Solana accounts do not require any on-chain initialization; once they contain
+some SOL, they exist. To set up a deposit account for your exchange, simply
+generate a Solana keypair using any of our [wallet tools](../wallet-guide/cli.md).
+
+We recommend using a unique deposit account for each of your users.
```

Comment:
We should mention that deposit accounts will be charged rent on first deposit and on epoch boundaries. Also, with eager rent collection, we will probably need a way to detect rent collections on accounts

Reply:
Related issue for listening to rent changes for accounts: https://github.com/solana-labs/solana/issues/10085

Reply:
I'll mention rent. I don't think rent is collected strictly on epoch boundaries, incidentally, but I don't think the current state of rent is documented anywhere :(
It would be nice to have to link to from this section.

Reply:
Ugh, how about just creating a placeholder doc for rent that you can link to here, then a github issue for filling it in later

Reply:
Rent seemed like a natural continuation from app/README.md, so I put it first under that section, but doesn't seem consistent with the other pages in that menu. Open to other suggestions.